package.json

```diff
@@ -20,5 +20,8 @@
     "e2e": "cd frontend; yarn e2e; cd ..",
     "spotless": "cd backend; ./gradlew spotlessApply; cd ..",
     "db": "docker-compose -f backend/docker-compose.yml exec db psql -U postgres simple_report"
+  },
+  "dependencies": {
+    "eslint-plugin-cypress": "^2.12.1"
```

Comment:
I don't think this needs to be added to the `package.json` in the root of the repo, it's already in `frontend/package.json`